doc/faq.rst

```diff
@@ -5,6 +5,19 @@
 Frequently Asked Questions
 ==========================
 
+Spicy Language
+--------------
+
+.. rubric:: Are Spicy's global variables *really* global?
+
+Indeed, they are. Changes to global variables become visible to all
+Spicy code; their values are not associated with specific connections
+or other dynamic state. This means that often globals won't be the
```

Comment:
Maybe mention here that they are global to a module.

Reply:
If they are public, they can be accessed from the outside as well. Is that what you mean?

Reply:
Unless that gets too technical adding that would help.

Is there anything else regarding globals one needs to think of when building a "base module" shared by multiple downstream modules?